validation/lagom-validation/user-management.md

```diff
@@ -0,0 +1,109 @@
+# User Validation
+Every invalid equivalence class returns an error. Every valid class returns an empty sequence.
+
+## Address
+
+### Street (String)
+*validCharactersStreet* := Letters ∪ Umlauts ∪ Digits ∪ Whitespace ∪ {ß , .  -}
+- vStreet1: street.length ∈ [1, 50] AND street has only valid characters
+- iStreet1: street.length ∉ [1, 50] 
+- iStreet2: street contains characters not defined in *validCharactersStreet*
+
+### houseNumber (String)
+Valid house numbers are of the form "42", "42b", and "53a-c". Letters may be lower- or uppercase.
+- vHouseNumber1: houseNumber is of one of the valid formats
+- iHouseNumber1: houseNumber is the empty String
+- iHouseNumber2: houseNumber has leading zero BUT is of one of a valid formats
+- iHouseNumber3: houseNumber is not of one of the valid formats
+
+### zipCode (String)
+- vZipCode1: zipCode is number with exactly five digits
+- iZipCode1: zipCode is a number with more or less than five digits
+- iZipCode2: zipCode is not a number
```

Comment:
we should change this to a number in the future then

Reply:
While we may be open for this in the future, the edge cases for leading zeros (if possible ?), all come from havon the zipCode represented as a number